main.go

```diff
@@ -104,6 +104,14 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Tobiko")
 		os.Exit(1)
 	}
+
+	performanceReconciler := &controllers.TrexPerformanceReconciler{}
+	performanceReconciler.Client = mgr.GetClient()
+	performanceReconciler.Scheme = mgr.GetScheme()
+	if err = tempestReconciler.SetupWithManager(mgr); err != nil {
```

Comment:
tempestReconciler -> performanceReconciler

Does it really work with this code as it is now? If yes, I'm little bit surprised. I'll need to test this manually.

Reply:
haven't got to actually testing it due to other priorities